services/search/src/search/duckdb_connection.py

```diff
@@ -5,6 +5,5 @@
 
 def duckdb_connect(**kwargs: Any) -> duckdb.DuckDBPyConnection:
     con = duckdb.connect(read_only=True, **kwargs)
-    con.sql("SET enable_external_access=false;")
```

Comment:
With the new version it throws the following error:
```
    # duckdb.duckdb.Error: Extension Autoloading Error:
    # An error occurred while trying to automatically install the required extension

```

Reply:
Doc says https://duckdb.org/docs/sql/configuration.html:
`Allow the database to access external state (through, e.g., loading/installing modules, COPY TO/FROM, CSV readers, pandas replacement scans, etc)`

It looks like duckdb now tries to dynamically load FTS extension from the db, so probably it makes sense to remove it.

Reply:
I found this [comment](https://github.com/huggingface/datasets-server/pull/1418#issuecomment-1742788076) cc. @albertvillanova @lhoestq

Reply:
As said in my comment, using `enable_external_access=false` should block HTTP requests (e.g. loading external data or extension) and avoid globbing local files to scan the local filesystem.

Is it possible to first load the FTS extension and then SET `enable_external_access=false` ?